frontend-react/src/oktaConfig.ts

```diff
@@ -13,15 +13,15 @@ const oktaAuthConfig: OktaAuthOptions = {
 };
 
 const oktaSignInConfig = {
-    logo: "https://reportstream.cdc.gov/assets/img/cdc-logo.svg",
+    logo: "/assets/cdc-logo.svg",
```

Comment:
Good catch since CSP may block if used from staging.
Are we sure we don't need a full path? e.g.
```
redirectUri: `${window.location.origin}/assets/cdc-logo.svg`,
```

Reply:
Pull the branch and give it a look, but it worked locally for me with the relative path :)